frontend/src/components/AdminControls/Users.js

```diff
@@ -1,133 +1,172 @@
-import React, { Fragment } from 'react';
+import React, { Component, Fragment } from 'react';
 import BootstrapTable from 'react-bootstrap-table-next';
 import paginationFactory from 'react-bootstrap-table2-paginator';
 import { Type } from 'react-bootstrap-table2-editor';
 import './AdminTables.css';
+import AdminEditModal from '../AdminEditModal/AdminEditModal';
 
-const Users = ({ users }) => {
-  const columns = [
-    {
-      dataField: 'inStock', // name?
-      text: '',
-      headerStyle: {
-        borderTop: 'none',
-        borderBottom: 'none',
-      },
-      style: {
-        paddingLeft: '10px',
-      },
-      formatter: (_cellContent, row) => (
-        <div className="checkbox">
-          <label htmlFor="adminCheckbox" className="admin-checkbox-container user-checkbox-spacing">
-            <input type="checkbox" id="adminCheckbox" checked={row.inStock} />
-            <span className="admin-table-checkmark" />
-          </label>
-        </div>
-      ),
-    },
-    {
-      dataField: 'full_name',
-      text: 'User Name',
-      style: { textAlign: 'center' },
-      headerStyle: {
-        textAlign: 'center',
-        borderTop: 'none',
-        borderBottom: 'none',
-      },
-    },
-    {
-      dataField: 'created_at',
-      text: 'Member Since',
-      style: { textAlign: 'center' },
-      headerStyle: {
-        textAlign: 'center',
-        borderTop: 'none',
-        borderBottom: 'none',
-      },
-      formatter: (cell) => {
-        let dateObj = cell;
-        if (typeof cell !== 'object') {
-          dateObj = new Date(cell);
-        }
+class Users extends Component {
```

Comment:
I mentioned this in person, but curious if we can manage this without changing this to a class-based component. I don't think it's a huge deal, but ideally we'd be moving in the opposite direction.

I think some of the stuff we're doing with state here either wouldn't need to be state or could be refactored pretty easily with the `useState` hook.